cmd/domains/main.go

```diff
@@ -263,3 +272,16 @@ func newPolicyService(cfg config, logger *slog.Logger) (policies.Service, error)
 
 	return policySvc, nil
 }
+
+func availableActionsAndBuiltInRoles(spicedbSchemaFile string) ([]roles.Action, map[roles.BuiltInRoleName][]roles.Action, error) {
```

Comment:
Rename to `listRoles` since that's what we are doing here. Actions are part of role.